### PR TITLE
Tickets/infra 219

### PR DESCRIPTION
**Pull request by hassanxp**

### CODE REVIEW

File: registries/princeton/documents.csv [hunk 1]
```diff
@@ -7,3 +7,26 @@
 "PFS-PFI-PRU040001-01", "Cobra Movement Test Report June 2018", "Erin Kado-Fong", "https://sumire.pbworks.com/w/file/fetch/126883949/PFS-PFI-PRU040001-01_CobraTests_June2018.pdf", "2018-06-08", 
 "PFS-ICS-PRU030000-01", "PFS ICS Hardware Configuration Report", "Hassan Siddiqui", "https://sumire.pbworks.com/w/file/126994364/PFS-ICS-PRU030000-01_PFS_ICS_Hardware_Configuration_Report.pdf", "2018-05-07", 
 "PFS-DRP-PRU050001-01", "Interim report on the status of 2D PSF modelling", "Neven Caplar", "https://sumire.pbworks.com/w/file/fetch/127999977/PFS-DRP-PRU050001-01_2D_modelling_status.pdf", "2018-06-26",
+"PFS-GEN-PRU000010-01", "Uploading Whitepaper HOWTO", "Jim Gunn", "", "2011-01-01", "howto dummies"
+"PFS-PFI-PRU000020-01", "Summary of Subaru/WFC Imaging Performance", "Jim Gunn", "", "2012-04-04", "Summary of Subaru/WFC imaging performance for PFS with vigetting, angles, etc."
+"PFS-PFI-PRU000030-01", "Notes on rotator and A&G cameras", "Jim Gunn", "", "2011-12-03", "Notes on the rotator and A&G cameras and rotation limits"
+"PFS-PFI-PRU000040-01", "Notes on geometry of fiber slit", "Jim Gunn", "", "2012-01-07", "Notes on the geometry of the fiber slit"
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(Fiber slit is a spectrograph component)

File: registries/princeton/documents.csv [hunk 2]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-PFI-PRU000050-01", "Notes on dither", "Jim Gunn", "", "2012-05-17", "Notes on the requirements for the dither mechanism for the FSA"
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(This paper discusses "slit" dithering, not dithering on the sky.)

File: registries/princeton/documents.csv [hunk 3]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-PFI-PRU000060-01", "Notes on aiming fibers", "Jim Gunn", "", "2013-09-12", "Notes on aiming the fibers to create a fiber pupil in the right place (on the gratings) AND the detailed geometry for the PFS slit masks.  Latest version (130912) has *hopefully* correct detector geometry and new slit curvature. PLEASE discard ALL earlier versions."
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(This paper discusses spectrograph design)

File: registries/princeton/documents.csv [hunk 4]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-PFI-PRU000070-01", "Notes on the formation of ghosts by glass slit plate", "Jim Gunn", "", "2012-09-10", "Notes on the formation of ghosts by a glass slit plate"
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(This paper discusses fiber slit design)

File: registries/princeton/documents.csv [hunk 5]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-PFI-PRU000080-01", "Fiber In/Out", "Jim Gunn", "", "2012-08-22", "Notes on aiming the fibers to create a fiber pupil in the right place (on the gratings) AND the detailed geometry for the PFS slit masks.  Latest version (130912) has *hopefully* correct detector geometry and new slit curvature. PLEASE discard ALL earlier versions."
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : FIB
(This paper discusses the entire fiber system. That is, not only Cobra-side, but also Spectrograph-side.)

File: registries/princeton/documents.csv [hunk 6]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-FIB-PRU000090-01", "Fiber FRD", "Jim Gunn", "", "2013-04-11", "Modelling the fiber FRD using the LNA experiments"
+"PFS-FIB-PRU000100-01", "Fiber Routing", "Jim Gunn", "", "2012-10-02", "Suggestion for fiber routing with no crossovers."
+"PFS-PFI-PRU000110-01", "PFS Focus", "Jim Gunn", "", "2012-11-18", "A study on the location of the PFS focal plane with respect to the WFC, and a short study on guiding, focussing, and aquisition errors."
+"PFS-FIB-PRU000120-01", "Crosstalk", "Jim Gunn", "", "2013-03-21", "A proposed definition of fiber-to-fiber crosstalk and a proposed specification"
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(This paper discusses spectrograph design.)

File: registries/princeton/documents.csv [hunk 7]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-PFI-PRU000130-01", "WFC Errors", "Jim Gunn", "", "2013-04-09", "A small study on the effect of the observed high-frequency slope errors in the WFC on Metrology Camera performance"
+"PFS-PFI-PRU000140-01", "Back Illumination", "Jim Gunn", "", "2013-04-16", "A small study on what illumination level is needed for the fiber back-illumination."
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : GEN?
Back illumination relates with both PFI (fiducial fibers) and SpS (science fibers), so I think "general" would be proper... what do you think of it?

File: registries/princeton/documents.csv [hunk 8]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-PFI-PRU000150-01", "Slit and Detectors", "Jim Gunn", "", "2013-09-11", "A document detailing the layout of the fibers along the fiber slit. Use in conjuction with the fiber routing white paper, which defines most of the nomenclature. This uses the detector geometry to define the slit geometry propagated to the newest version of the fiber aiming paper. (revised 130911 to include the medium resolution mode footprints and the new slit curvature value (.00151/mm))."
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(This discusses spectrograph design)

File: registries/princeton/documents.csv [hunk 9]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-PFI-PRU000160-01", "Slit Curvature", "Jim Gunn", "", "2013-08-07", "A small study on the effects of slit curvature on image performance metrics in the presence of FRD"
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(This discusses spectrograph design)

File: registries/princeton/documents.csv [hunk 10]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-SPS-PRU000170-01", "Cooling IR4", "Jim Gunn", "", "2014-09-26", "A study of cooling requirements for the SCR and spectrograph modules, and a proposed plumbing layout"
+"PFS-PFI-PRU000180-01", "Calibration Design", "Jim Gunn", "", "2014-10-15", "A very preliminary  document describing the calibration system mounted on top of the PFI"
+"PFS-PFI-PRU000190-01", "AG Focus", "Jim Gunn", "", "2015-05-18", "Study of AG camera depth, number of stars, and focus servo"
+"PFS-PFI-PRU000200-01", "Slit 2 FRD", "Jim Gunn", "", "2015-12-07", "Evaluation of LNA Slit 2 performance (without cover plate)"
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(This discusses analysis of the fiber slit test)

File: registries/princeton/documents.csv [hunk 11]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-FIB-PRU000210-01", "Fiber Connectors", "Jim Gunn", "", "2016-03-28", "Study of elastic properties of mating MTP multifiber connectors"
+"PFS-PFI-PRU000220-01", "IR Thermal Background", "Jim Gunn", "", "2017-08-02", "The Thermal Background in the PFS IR Dewar"
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : SPS
(This discusses NIR camera design.)

File: registries/princeton/documents.csv [hunk 12]
```diff
@@ -7,3 +7,26 @@
...
+"PFS-PFI-PRU000230-01", "COBRA and FIDUCIAL Coordinates and Numbering Scheme", "Jim Gunn", "", "2017-11-14", "The COBRA and FIDUCIAL Coordinates and Numbering Scheme"
```
Comment (yukimoritani) [inline registries/princeton/documents.csv]: Category : FIB
(This discusses fiber mapping.)

**Review by yukimoritani — CHANGES_REQUESTED** [review]
I'd like to suggest proper categories for some documents.

**Comment by hassanxp** [comment]
I have implemented all your suggested changes. Ok to merge the PR?

**Review by yukimoritani — APPROVED** [review]

**Pull request merged**